tjreports/site/assets/js/tjreports.js

```diff
@@ -384,6 +384,12 @@ function getReportdata(page, colToShow, limit, sortCol, sortOrder, action, allow
 		filterValue = techjoomla.jQuery(this).val();
 		filterName = techjoomla.jQuery(this).attr('id');
 		filterName = filterName.replace('search-filter-','');
+		
+		if (filterName == 'userType')
```

Comment:
1. if (filterValue != '' && ifInColsToShow != -1)
2. The above condition is used to insert the data into filter array.
3. If the column name is checked ( var isChecked = techjoomla.jQuery(this).is(":checked") ) then only we can set filter for it, but "userType is not a column" so the above condition will always false.